modules/apps/commerce/commerce-price-list-api/src/main/java/com/liferay/commerce/price/list/exception/CommerceTierPriceEntryQuantityException.java

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.commerce.price.list.exception;
+
+import com.liferay.portal.kernel.exception.PortalException;
+
+/**
+ * @author João Victor Cordeiro
+ */
+public class CommerceTierPriceEntryQuantityException
```

Comment:
I see that this one has been added manually, without going through
modules/apps/commerce/commerce-price-list-service/service.xml
is there a reason for that?

Reply:
hi @luca-pellizzon , I can take care of it. I was not aware we do exception generations through service.xml now.

Reply:
@luca-pellizzon , the author name can be changed manually yes? I generated the actual file, but it still gives author name for Alessio.

Reply:
don't need to change the author when it is autogenerated

Reply:
So we do commit the exception file with author as Alessio then? when I run buildService the Exception gets generated with him as the author.

Reply:
@luca-pellizzon , the changes should be satisfactory now.